### ISSUE TITLE
Core style: All links on single listing should either have underline or icon  

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#3984]
**Describe the bug**
All links should have a visual indication other than just color for accessibility.

**To Reproduce**
1. Go to single listing page

**Expected behavior**
The only link with color only right now that I see is the "View on Map" link  which should receive an underline. We can also discuss if its simpler to add underline to all text links on the page which I would be open to.

Expected style:
https://www.figma.com/file/3AE2eZo0oq7kSVNgb0XHoC/Combined-Visual-QA?type=design&node-id=24%3A1878&mode=design&t=pGUJSWma3JdmIgIo-1

**Screenshots**
Current state:
https://www.figma.com/file/3AE2eZo0oq7kSVNgb0XHoC/Combined-Visual-QA?type=design&node-id=24%3A429&mode=design&t=pGUJSWma3JdmIgIo-1
